### PR TITLE
Updates for XGBoost / FIL examples

### PR DESCRIPTION
**Pull request by drobison00**
Updates required to support additional workflows, custom mortgage data, and FIL inference.

### CODE REVIEW

File: dask/kubernetes/specs/worker-spec-custom.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        items:
+          - key: keyfile.json
+            path: keyfile.json
+  tolerations:
+    - key: "daskrole"
+      operator: "Equal"
+      value: "worker"
+      effect: "NoSchedule"
+  containers:
+    - image: gcr.io/[YOUR_GCP_PROJECT]/[YOUR_USER_ID]/dask-unified:0.18
```
Comment (zronaghi) [inline dask/kubernetes/specs/worker-spec-custom.yaml:20]: Can we update the containers to 0.19? And the Dockerfile in this folder

**Review by zronaghi — CHANGES_REQUESTED** [review]

**Review by Nanthini10 — APPROVED** [review]
Looks good to me, I just have one suggestion

File: dask/kubernetes/specs/sched-spec.yaml [hunk 2]
```diff
@@ -5,15 +5,15 @@ metadata:
...
-    - image: gcr.io/[YOUR_PROJECT_PATH]/dask-unified:0.18
+    - image: gcr.io/[YOUR_GCP_PROJECT]/[YOUR_USER_ID]/dask-unified:0.19
       imagePullPolicy: IfNotPresent
       args: [ dask-scheduler ]
       name: dask-scheduler
       resources:
         limits:
-          cpu: "3"
-          memory: 10G
+          cpu: "10"
```
[the remark this answers is not in the document]
Comment (drobison00) [reply dask/kubernetes/specs/sched-spec.yaml:14]: That's a good point. I think these moved around a bit while I was testing on different platforms. I'll go back and set them up so they're targeted at n1-standard-4 machines.

**Comment by zronaghi** [comment]
Looks good, will merge once the default hardware gets updated

**Review by zronaghi — APPROVED** [review]
I'm going to merge this and @Nanthini10 will modify the default params (thanks!)

**Pull request merged**